### ISSUE TITLE
Light card color issue

### ISSUE THREAD

**Issue by Dae657** [Madelena/Metrology-for-Hass#63]
The color seems to have become much more hard to see on my light cards, especially as compared to yours. If I look very closely when moving sliders, I can see where it would be.

![Screenshot (2)](https://user-images.githubusercontent.com/106887758/225806144-513ff206-8847-4804-82e3-a5f23dd80273.png)

 If I remove the CSS styling for slider-bg from the light card, it comes back fully.

![Screenshot (4)](https://user-images.githubusercontent.com/106887758/225810379-adf366db-95bb-4274-a761-6baec25b770b.png)

I also changed the toggle_on_click to false, and when you use the slider, a bright line appears at the slider's edge with the CSS styling as-is. I don't see this as a problem, just in case it helps pinpoint the origin.

![Screenshot (3)](https://user-images.githubusercontent.com/106887758/225809735-cc288952-587f-4e7f-a655-8ca6fb153f2b.png)

Is there a way to change slider_bg opacity to fix that so it looks more like yours?

To that end, my light group cards (which I know are a different template) just don't show that color at all, not even on the bottom trim. If I remove the CSS styling on that, it just gives me the full metro purple (in this case) theme color card.

Separately, is there a way I can get rid of the glowing shadow box around the dialog boxes? I love the theme otherwise

**Comment by xbouthoorn** [comment]
@Dae657 how did you fix this? I'm facing the same issue.

**Comment by Dae657** [comment]
@xbouthoorn  Hm, that didn't change anything for me. Definitely glad you got it working for yours though!

I did notice that she mentioned on the reddit post for it that she was using the fluent pink (now red?) theme - it works with a dark fluent (purple) theme, but not metro, not sure if that's the difference.  I'm using metro for the black background; I might try to create a color theme as she gives the option to at the end of the metro.yaml with mostly fluent colors, but a black background as a workaround to see if that works.

**Comment by Madelena** [comment]
This is probably due to a previous fix for #26 changing the `ha-card-background` color from transparent to opaque.

To fix this, I added this to `light-card` in `decluttering-card.yaml`:
```yaml
      .slider {
        background: linear-gradient(to right, rgba(var(--rgb-card-background-color),0.8), rgba(var(--rgb-card-background-color),0.8)), linear-gradient(to right, var(--slider-color), var(--slider-color));
      }
```
